content/docs/app-developer-guide/export-to-oci-layout.md

```diff
@@ -0,0 +1,151 @@
++++
+title="Export to OCI layout format"
+weight=3
+summary="Learn how to export your application image to disk in OCI layout format"
++++
+
+<div class="quote mb-4">
+    The OCI Image Layout is the directory structure for OCI content-addressable blobs and location-addressable references.
+    <div class="author">See the <a href="https://github.com/opencontainers/image-spec/blob/main/image-layout.md">specification</a></div>
+</div>
+
+Exporting to OCI layout format is an **experimental** feature available on pack since version X.Y.Z
+
+### 1. Enable experimental feature
+
+Verify your pack version is equal or greater than X.Y.Z
+
+```bash
+pack version
+```
+
+Enable the experimental features on pack
+
+```bash
+pack config experimental true
+```
+
+You can confirm everything is fine, checking the `config.toml` file in your `PACK_HOME` installation folder, for example:
+
+```bash
+cat ~/.pack/config.toml
+experimental = true
+layout-repo-dir = "<$HOME>/.pack/layout-repo"
```

Comment:
Does `pack config experimental true` add the "layout-repo-dir" to `config.toml`?

Reply:
Yes, that's correct